examples/tutorial/napa-runner/main.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#include <napa.h>
+
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+void ReadJSFromFile(std::stringstream& jsStream, const std::string& jsFileName);
+
+int main(int argc, char* argv[])
+{
+    if (argc != 2) {
+        std::cout << "usage: napa-runner <js-file>" << std::endl;
+        exit(0);
+    }
+
+    // Read js from user specifed js file.
+    std::stringstream jsStream;
+    ReadJSFromFile(jsStream, argv[1]);
+
+    // Initializes napa with glabal scope settings.
+    napa::Initialize();
+
+    // Create a napa zone with 1 worker.
+    auto zone = std::make_unique<napa::Zone>("z1", "--workers\t1");
```

Comment:
how about use 'main' zone to be more formal? In case people want to use this as a tool or copy/paste code :-)